c2rust-ast-exporter/build.rs

```diff
@@ -150,6 +150,15 @@ fn build_native(llvm_info: &LLVMInfo) {
     };
 
     // Statically link against 'clangAstExporter' which requires 'tinycbor'
+    // Use a custom tinybor directory via an environment variable
+    if let Ok(tinycbor_dir) = env::var("C2RUST_TINYCBOR_PATH") {
```

Comment:
From what I've seen (correct me if I'm wrong), `TINYCBOR_DIR` would be more idiomatic for the environment variable here.

Reply:
@kkysen I don't think you're wrong, and I even originally had `TINYCBOR_PATH`. I was just worried that it would be more likely to clash, which probably isn't a real concern. I can change this.